pymc3/distributions/__init__.py

```diff
@@ -24,6 +24,7 @@
 from .continuous import ExGaussian
 from .continuous import VonMises
 from .continuous import SkewNormal
+from .continuous import Triangular
```

Comment:
you'll need to add `'Triangular'` to the `__all__` list in this file as well -- should fix the test error.